CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Changelog 
```

Comment:
This is what https://github.com/xdslproject/xdsl/blob/main/.github/release-drafter.yml
is meant to automatically work on.
PLease add a tag/label to your PR and every PR from now on.

Reply:
Oh okay, I now understand, thanks!